ktxspec.adoc

```diff
@@ -671,6 +675,9 @@ transcode target format is selected.
 ====
 
 ===== Zstandard
+* After inflation, the level data follows the uncompressed layout as
+  specified in the <<levelImages>> with <<_mippadding,mipPadding>> omitted.
```

Comment:
Currently when inflating a ktxTexture2, `libktx` inserts the same padding as the data would have had before deflation. There is no mention of this topic in the linked issue though I vaguely recall discussing it somewhere.

What is the reason padding should be omitted? If the image format has multi-byte data then you want it properly aligned for uploading to the GPU.

Reply:
You're right. I was confused by the following language from the Section 2:
> Applications inflating levels may choose to restore the alignment caused by `mipPadding`.